README.md

```diff
@@ -48,19 +48,6 @@ HOST: http://example.com
 `) // 'text/vnd.apiblueprint'
 ```
 
-## Documentation
-
-### identify (function)
-
-```javascript
-/**
- * Identifies given source.
- * @param {string} source - The source code of API description file.
- * @returns {string|null} Media type of given file.
- */
-deckardcain.identify(source)
-```
-
```

Comment:
I don't get what you mean. I removed this part as it is just one-file, heavily commented library and reading the code instead of proper documentation is not such an issue.